packages/myst-directives/src/admonition.ts

```diff
@@ -4,6 +4,7 @@ import { ParseTypesEnum } from 'myst-common';
 
 export const admonitionDirective: DirectiveSpec = {
   name: 'admonition',
+  doc: 'Callouts, or "admonitions", highlight a particular block of text that exists slightly apart from the narrative of your page, such as a note or a warning. \n\n The admonition kind can be determined by the directive name used (e.g. `:::{tip}` or `:::{note}`).',
```

Comment:
I have added a few places of better documentation to the various directives. I will continue improving that a bit in this PR!